### PR TITLE
unset production environment for archived branch v0.35

### PR DESCRIPTION
**Pull request by vjjmiras**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind bug

> /kind cleanup

> /kind design

> /kind user-interface

> /kind content

> /kind translation

> /kind event

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area blog

/area documentation

> /area community

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

#1172 for branch v0.35

**Special notes for your reviewer**:

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -14,10 +14,12 @@ HUGO_ENV = "production"
 HUGO_ENABLEGITINFO = "true"
 
 [context.deploy-preview]
+HUGO_ENV = "development"
 command = "make preview-build"
 
 [context.branch-deploy]
-command = "make preview-build"
+HUGO_ENV = "archive"
```
Comment (LucaGuerra) [inline netlify.toml:21]: It looks like it's working! Thank you :tada:  Is this something one needs to set every time a branch is archived? Is it the only thing? If so we should simply document it

Comment (LucaGuerra) [reply netlify.toml:21]: Answering myself, it does not look like it, because this is specific for branches. That's good!

Comment (vjjmiras) [reply netlify.toml:21]: That's right, it'll transparently unmark the archived versions as production. No extra steps are required unless future updates of Docsy change that again.

**Review by Issif — APPROVED** [review]

**Pull request merged**